dnf/cli/cli.py

```diff
@@ -803,6 +803,11 @@ def configure(self, args, option_parser=None):
             e = '%s: %s' % (ucd(e.args[1]), repr(e.filename))
             logger.critical(_('Config error: %s'), e)
             sys.exit(1)
+        if opts.destdir is not None:
+            self.base.conf.destdir = opts.destdir
+            if self.base.conf.downloadonly or opts.command == 'download':
+                logger.critical(_('--destdir must be used with --downloadonly or download command. Exiting.'))
```

Comment:
I think that splitting this would be counterproductive and would bring no real benefit except conforming to the rules set by the linter just because they exist.

PEP8 sets the limit to 79, probably because terminals default to 80 characters per line. I do not know why DNF has set the max line length to 100, however.

If someone with a narrow monitor tries to read this, it will probably wrap, which is no problem for a line consisting mainly of a string.